### PR TITLE
Added Watch later and Likes functionality

### PR DESCRIPTION
**Pull request by surbhi84**
This pr consists of:

- Like managment and watch later management.
- You can add videos to watch later page.
- You can add remove videos using the same button.
- Some other changes for design improvement and a delete all history button.

https://tapesvideolibrary.netlify.app/
check here

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -36,3 +39,39 @@ export function deleteHistory(id, encodedToken) {
...
+export function postWatchLater(video, encodedToken) {
+  return axios.post(
+    "/api/user/watchlater/",
+    { video },
+    { headers: { authorization: encodedToken } }
+  );
+}
+
+export function deleteWatchLater(id, encodedToken) {
+  return axios.delete("/api/user/watchlater/" + id, {
```
Comment (ishitakeshawani) [inline src/apiCalls.js:74]: suggestion: you can use template literals here.

**Review by ishitakeshawani — APPROVED** [review]
Liked toasts.
comments are helpful in code reading.
left a suggetsion.

**Comment by ishitakeshawani** [comment]
Deploy preview showing 404.
image

File: src/hooks/reducer/userReducer/reducer.js [hunk 2]
```diff
@@ -31,25 +34,55 @@ export const useUserReducer = () => {
...
+        return {
+          ...state,
+          user: {
+            ...state.user,
+            watchLater: [...state.user.watchLater, payload],
+          },
+        };
+      }
+
+      case DELWATCHLATER: {
```
Comment (viru2001) [inline src/hooks/reducer/userReducer/reducer.js:80]: you can add a separator between 2 words for e.g.
ADD_WATCH_LATER

**Review by viru2001 — APPROVED** [review]
- code is clean
- good folder structure
- nice use of tailwind CSS

left one suggestion on the naming of case in Reducer function so that it will be more readable

**Pull request merged**